docs/reference/targets.md

```diff
@@ -121,4 +121,15 @@ with a matching name, the corresponding callback _will not_ be invoked again.
 
 ## Naming Conventions
 
-Always use camelCase to specify target names, since they map directly to properties on your controller.
+Always use camelCase to specify target names, since they map directly to properties on your controller:
+
+```erb
```

Comment:
maybe make this `html` as not everyone who uses stimulus uses ruby.

Also use HTML syntax

Reply:
Oops. Nice catch.